src/apscheduler/_schedulers/async_.py

```diff
@@ -529,6 +532,29 @@ async def remove_schedule(self, id: str) -> None:
         self._check_initialized()
         await self.data_store.remove_schedules({id})
 
+    async def pause_schedule(self, id: str) -> None:
```

Comment:
The `id` parameter shadows the built-in `id()` function. You should rename it to `schedule_id`.

Reply:
@agronholm For this I was just following what was used by other functions, since consistency is valuable for parts of the external API like this. Would you still rather I use `schedule_id` even though the other similar methods use `id`?

Reply:
In 61f7051 I've updated `pause_schedule` and `unpause_schedule` to use `schedule_id` instead of `id`. If you'd rather `id` be used consistently, I can revert that change.

Reply:
You're right, I've been using `id` for other such parameter names. It's okay to revert.